TMessagesProj/src/main/java/org/telegram/ui/Cells/UserCell2.java

```diff
@@ -291,6 +291,7 @@ public void update(int mask) {
             imageView.setVisibility(currentDrawable == 0 ? GONE : VISIBLE);
             imageView.setImageResource(currentDrawable);
         }
+        if(currentUser!=null &&currentUser.id !=UserConfig.getInstance(currentAccount).getClientUserId() &&currentUser.mutual_contact) statusTextView.setText(statusTextView.getText()+" mutual contact");
```

Comment:
not a proper way, better will be to create a new cell that will be shown only if contact is mutual.